Modules/Core/Common/include/itkTriangleCell.hxx

```diff
@@ -237,12 +238,7 @@ TriangleCell<TCellInterface>::DistanceToLine(PointType   x,
     denom += static_cast<double>(v21[i] * v21[i]);
   }
 
-  // trying to avoid an expensive fabs
-  double tolerance = 1.e-05 * num;
-  if (tolerance < 0.0)
-  {
-    tolerance = -tolerance;
-  }
+  double tolerance = std::abs(1.e-05 * num);
```

Comment:
The comment "trying to avoid an expensive fabs" appears to be introduced by commit cc69c19e6094ce426c9abdcf0a9756b650b11c93 "ENH: Added EvaluatePosition()...", Julien Jomier (@jjomier), Nov 16, 2004. `fabs` might have been expensive twenty years ago. But nowadays, it appears to be the other way around! When compiler optimization is enabled, `fabs` appears to be compiled to just a single machine instruction! While the manual approach (`if (x < 0)...`) appears to yield quite a few more machine instructions! Example at https://godbolt.org/z/M8e6x8ssh

```cpp
double Call_fabs(double x)
{
    return fabs(x);
}

double Call_std_abs(double x)
{
    return std::abs(x);
}

double ManuallyEstimateAbs(double x)
{
    if (x < 0)
    {
        return -x;
    }
    return x;
}
```

Output:
```asm
Call_fabs(double):
        andpd   xmm0, XMMWORD PTR .LC0[rip]
        ret
Call_std_abs(double):
        andpd   xmm0, XMMWORD PTR .LC0[rip]
        ret
ManuallyEstimateAbs(double):
        pxor    xmm2, xmm2
        movapd  xmm3, xmm0
        movapd  xmm1, xmm0
        xorpd   xmm1, XMMWORD PTR .LC2[rip]
        cmpltsd xmm0, xmm2
        andpd   xmm1, xmm0
        andnpd  xmm0, xmm3
        orpd    xmm0, xmm1
        ret
.LC0:
        .long   -1
        .long   2147483647
        .long   0
        .long   0
.LC2:
        .long   0
        .long   -2147483648
        .long   0
        .long   0
```